src/client/components/report_category.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+import React, { Component } from "react";
+import styled from "styled-components";
+import Select from '@material-ui/core/Select';
+import { Link } from 'react-router-dom';
```

Comment:
If we don't use the statement make sure to do clean up so we don;t have an overly large js file